docs/database-connection-string.md

```diff
@@ -1,85 +1,83 @@
+The database connection string is a character string that is used to define and configure the connection between an application and a Starcounter database. It consists of a number of option assignments, separated by semicolons (see [sample](#connection-string-sample)).
 
-## Connection String options
+## Connection string options
 
-| Name | Type | Required | Default Value | Description |
-| :--- | :--- | :--- | :--- | :--- |
-| `Database` | `string` | True | None | Absolute or relative database directory path. |
-| `OpenMode` | `enum` | False | `CreateIfNotExists` | Specifies database creation strategy. |
-| `StartMode` | `enum` | False | `StartIfNotRunning` | Specifies database startup strategy. |
-| `StopMode` | `enum` | False | `IfWeStarted` | Specifies database shut down strategy. |
-| `ContextCount` | `int` | False | 2 - 24 | Specifies number of database contexts allocated for this connection. |
+Name         | Type      | Required | Default value       | Description
+:----------- | :-------- | :------- | :------------------ | :-----------------------------------------------------------------------
+Database     | `string`  | Yes      | None                | A path string defining the database to use.
+OpenMode     | `enum`    | No       | `CreateIfNotExists` | Specifies the database creation strategy.
+StartMode    | `enum`    | No       | `StartIfNotRunning` | Specifies the database startup strategy.
+StopMode     | `enum`    | No       | `IfWeStarted`       | Specifies the database shut down strategy.
+ContextCount | `integer` | No       | `2` – `24`          | Specifies the number of database contexts allocated for this connection.
 
-### The `Database` value
+The options are further explored in the sections below.
 
-Starcounter database is identified by it's physical location and Starcounter version which attempts to access it.
+### Database
 
-For example, attempting to start a database located at `/home/databases/TestDatabase` with Starcounter version `3.0.0-00001` will do the following:
+The Database option is required in the connection string, and consists of an absolute or relative path to the directory where the Starcounter database files are located, or should be located if they do not yet exist.
 
-- Calculate database identification: `Starcounter-3.0.0-00001-/home/databases/TestDatabase`.
-- Check if there is a running `scdata` process associated with this database identification.
-- If no: spin up a new `scdata` process. This will fail if there is another version of `scdata` process already running for this database.
-- Communicate with the `scdata` process.
+Internally, a Starcounter database is identified by it's physical location on disk and the version of Starcounter that is trying to access it. For example, attempting to start a database located at `/home/databases/TestDatabase` with Starcounter version `3.0.0-00001` will lead to this behavior:
```

Comment:
>... it's physical location on disk ...

Should not it be `its`?

Also, the database is not identified by its physical location, but rather by the path provided in the connection string. If there would be a database located at `/path/db` and an environment variable called `path_db`, then two apps try to connect, one provides `/path/db` other one provides `%path_db%`. Starcounter will not detect that it is the same database, and will try to spin two `scdata` processes. Second attempt will fail, because the database files will be locked by the first `scdata` process.

Reply:
It's already covered [here](https://github.com/Starcounter/Docs/pull/271/files#diff-cfb400e2cbd66e097e7a0f52bb00e03dR33), but the original description is a bit misleading, I think.

Reply:
> Should not it be its?
Yes it should!